lib/menu.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'rice'
+require_relative 'chicken'
+require_relative 'lamb'
+require_relative 'mutton'
+require_relative 'naan'
+require_relative 'poppadom'
+# require_relative 'message'
+
+class Menu
+
+	attr_reader :items, :order, :total
+
+	def initialize
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for initialize is too high. [18.25/15]